### ISSUE TITLE
Needs local debug

### ISSUE THREAD

**Issue by vsoch** [spack/spack-stack-catalog#7]
The automated update started failing a few days ago with some issue with the query.

**Comment by vsoch** [comment]
And it looks like the code search is empty - I'll take a closer look today! I suspect the API changed.

```
code_search = g.search_code("spack filename:spack.yaml+-user:vsoch", sort="indexed")
code_search.totalCount
Out[7]: 0
```

**Comment by vsoch** [comment]
@tgamblin any particular reason I've been denied any write access to my own project?

**Comment by tgamblin** [comment]
@vsoch I took you off the Spack org because I didn't think you wanted to be on it.  I've added you as admin here.

**Comment by vsoch** [comment]
Awesome - much appreciated! I'm good to keep maintaining the projects that I explicitly created, at least that is the responsible thing to do I think.

**Comment by vsoch** [comment]
Especially the ones with pancake puns, those are extra special :pancakes:

**Comment by vsoch** [comment]
We are in business!

![image](https://github.com/spack/spack-stack-catalog/assets/814322/37ff42cb-48a4-4096-8de2-3bd242c24ebf)

The bug was a change to the structure of the search query - previous queries worked with adding `+` between parameters, but that seems to be changed to expect spaces (and the `+` delimiter will return 0 results). Assuming this could happen again, I added an exception raise given the case of 0 results, so at least next time I'll see it.

Closing issue. Thanks again @tgamblin.
